src/components/CodeChanges/CodeChanges.tsx

```diff
@@ -114,12 +118,11 @@ const StyledButton = styled(Button)`
   margin-right: ${size.xs};
 `;
 
-// @ts-expect-error
-const Title: SubtitleType = styled(Subtitle)`
+const Title = styled(Subtitle)`
   font-weight: normal;
   margin-right: ${size.s};
   margin-bottom: ${size.s};
-`;
+` as StyledComponent<SubtitleProps>;
```

Comment:
I'd be curious to understand why you went with this approach as opposed to something like this. Does the type assertion offer any advantages?

```
const Title = styled(Subtitle)<SubtitleProps>`
  font-weight: normal;
  margin-right: ${size.s};
  margin-bottom: ${size.s};
`;
```

Reply:
I went with this approach because it's what LeafyGreen recommended for this but I actually do agree this is nicer looking. Updated!